test/integration_test.go

```diff
@@ -101,10 +121,32 @@ func prettyPrintWhoCanOutput(t *testing.T, args []string, out *bytes.Buffer) {
 	}
 }
 
-func configureRBAC(t *testing.T, client kubernetes.Interface) {
+func createCRDs(t *testing.T, client clientext.CustomResourceDefinitionInterface) {
 	t.Helper()
+	_, err := client.Create(&apiext.CustomResourceDefinition{
+		ObjectMeta: meta.ObjectMeta{
+			Name: "pods.metrics.k8s.io",
+		},
+		Spec: apiext.CustomResourceDefinitionSpec{
+			Scope: apiext.NamespaceScoped,
+			Group: "metrics.k8s.io",
+			// TODO Use Versions filed to be a good citizen
```

Comment:
TODO here (which I don't totally understand!)

Reply:
My bad. The Version property is deprecated. I've changed it to:

```go
Versions: []apiext.CustomResourceDefinitionVersion{
	{
		Name:    "v1beta1",
		Served:  true,
		Storage: true,
	},
},
```